### PR TITLE
Tell users to accept caseless input for usernames

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Fixes [#2055](https://github.com/alphagov/govuk-design-system/issues/2055).

This PR updates our 'Create a username' pattern.

### CODE REVIEW

File: src/patterns/create-a-username/index.md.njk [hunk 1]
```diff
@@ -30,13 +30,14 @@ You should do research to understand this group and have a plan for helping them
...
 
 You need to:
 
-- tell people whether their proposed username is unique
-- suggest unique ones for them, in some cases
-- make sure people can retrieve or reset their username
+- tell the user whether their proposed username is unique
+- suggest a unique username for them, in some cases
+- make sure the user can retrieve or reset their username
+- accept usernames caselessly - for example, if the actual username is `‘Mary@example.com’`, the user can still sign in with `‘mary@example.com’`
```
Comment (EoinShaughnessy) [inline src/patterns/create-a-username/index.md.njk:40]:
```suggestion
- accept usernames caselessly - for example, if the actual username is ‘Mary@example.com’, the user can still sign in with ‘mary@example.com’
```

File: src/patterns/create-a-username/index.md.njk [hunk 2]
```diff
@@ -30,13 +30,14 @@ You should do research to understand this group and have a plan for helping them
...
 
 You need to:
 
-- tell people whether their proposed username is unique
-- suggest unique ones for them, in some cases
-- make sure people can retrieve or reset their username
+- tell the user whether their proposed username is unique
+- suggest a unique username for them, in some cases
+- make sure the user can retrieve or reset their username
+- accept usernames caselessly - for example, if the actual username is ‘Mary@example.com’, the user can still sign in with ‘mary@example.com’
```
Comment (EoinShaughnessy) [inline src/patterns/create-a-username/index.md.njk:40]: Any idea for how to display 'Mary/mary@example.com' in ordinary text?

In the preview, they both display as hyperlinks. I then tried putting backticks around them, but that made them display in code formatting.

Comment (calvin-lau-sig7) [reply src/patterns/create-a-username/index.md.njk:40]: I looked in the email address pattern and it used this odd trick of putting an HTML tag in the middle of it.

Not sure if 'caselessly' is a good word? Tried to break it down but not sure 'regardless' is quite plain language enough either. Only other alternative I can think of so far is: "ignore letter case when accepting usernames" 

```suggestion
- accept usernames regardless letter case — for example, if the actual username is ‘Mary<i></i>@example.com’, the user can still sign in with ‘mary<i></i>@example.com’
```

Comment (EoinShaughnessy) [reply src/patterns/create-a-username/index.md.njk:40]: 'Regardless' is definitely better than 'caselessly', but yeah, ain't the plainest for sure. Your other alternative sounds great (imo). And cheers for sharing the trick with the HTML tag! 👍🏻 

```suggestion
- ignore letter case when accepting usernames - for example, if the actual username is ‘Mary<i></i>@example.com’, the user can still sign in with ‘mary<i></i>@example.com’
```

**Review by calvin-lau-sig7 — APPROVED** [review]

**Pull request merged**